yiisoft/yii2-bootstrap/CVE-2015-3397.yaml

```diff
@@ -0,0 +1,8 @@
+title:     JSON Data encoded for use in HTML was not safe to use in IE6/IE7, possible XSS attacks
+link:      http://www.yiiframework.com/news/86/yii-2-0-4-is-released/
+cve:       CVE-2015-3397
+branches:
+    master:
```

Comment:
It does, as it seems: it is hip nowadays! :D

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

On 13 May 2015 at 14:25, Christophe Coevoet <notifications@github.com>
wrote:

> In yiisoft/yii2-bootstrap/CVE-2015-3397.yaml
> <https://github.com/FriendsOfPHP/security-advisories/pull/66#discussion_r30229548>
> :
>
> > @@ -0,0 +1,8 @@
> > +title:     JSON Data encoded for use in HTML was not safe to use in IE6/IE7, possible XSS attacks
> > +link:      http://www.yiiframework.com/news/86/yii-2-0-4-is-released/
> > +cve:       CVE-2015-3397
> > +branches:
> > +    master:
>
> @reiz <https://github.com/reiz> does versioneye import advisories posted
> in this repo automatically ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FriendsOfPHP/security-advisories/pull/66/files#r30229548>
> .
>